test/js/node/child_process/child_process-node.test.js

```diff
@@ -397,38 +398,44 @@ describe("child_process double pipe", () => {
     // const { mustCallAtLeast, mustCall } = createCallCheckCtx(done);
     const mustCallAtLeast = fn => fn;
     const mustCall = fn => fn;
-    let grep, sed, echo;
-    grep = spawn("grep", ["o"], { stdio: ["pipe", "pipe", "pipe"] });
-    sed = spawn("sed", ["s/o/O/"]);
-    echo = spawn("echo", ["hello\nnode\nand\nworld\n"]);
+    let fakeGrep, fakeSed, fakeEcho;
+    fakeGrep = spawn(bunExe(), [
+      "-e",
```

Comment:
Fake grep, sed, and echo with bun because the windows versions work differently.